docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -247,15 +247,17 @@ Physical nodes (or cloud instances) are installed with a standard Centos 7 serve
     sudo chronyc makestep   # (force instant sync to NTP server)
     ```
 
-1. Add a new `yugabyte:yugabyte` user and group (sudo is required):
+1. Add a new `yugabyte:yugabyte` user and group via the default login shell `/bin/bash` prepended with the `-s` flag (sudo is required):
```

Comment:
Thing "via" is misleading since you can create the user from any shell, we are just setting the yugabyte user's default login shell to "/bin/bash".

Reply:
Add a new `yugabyte:yugabyte` user and group with the default login shell (set via `-s` flag) `/bin/bash` (sudo is required)